cmd/fluxctl/install_cmd.go

```diff
@@ -44,8 +44,16 @@ fluxctl install --git-url 'git@github.com:<your username>/flux-get-started' | ku
 		"Email to use as git committer")
 	cmd.Flags().StringVar(&opts.Namespace, "namespace", getKubeConfigContextNamespace("default"),
 		"Cluster namespace where to install flux")
+	cmd.Flags().StringVar(&opts.Image, "flux-image", "",
+		"Image the Flux daemon container should run, defaults to the latest release for this fluxctl version")
```

Comment:
Maybe add an example of the format? (Because it's not clear whether the tag should be included

Reply:
I assume most people don't know what a FQIN is, or do they?